.github/workflows/codeql-analysis.yml

```diff
@@ -0,0 +1,67 @@
+# For most projects, this workflow file will not need changing; you simply need
+# to commit it to your repository.
+#
+# You may wish to alter this file to override the set of languages analyzed,
+# or to provide custom queries or build logic.
+#
+# ******** NOTE ********
+# We have attempted to detect the languages in your repository. Please check
+# the `language` matrix defined below to confirm you have the correct set of
+# supported CodeQL languages.
+#
+name: "CodeQL"
+
+on:
+  push:
+    branches: [ master ]
```

Comment:
Wouldn't we want to run this on all branches, i.e. also in forks?

Reply:
I think that's right. Although, when a PR is open from a fork to push to master, this analysis will run. 

Curious, are actions (workflows) currently running in your forks?

Reply:
Checking on this, I think the workflows are executing your meshery fork today and that it might be a similar configuration to invoke on push to master is running in the remote fork (here's an example - https://github.com/layer5io/meshery/blob/master/.github/workflows/build-and-release.yml#L7).

Yes, having this build on any branch sounds good. My ignorance gets in the way of being able to intelligible supporting statements to this suggestion as well as intelligible detractors. @ramrodo, thoughts?

Reply:
Yes, when we update our fork from the latest code of upstream. The actions do run on our forks as well.

Reply:
Here's another example: https://github.com/layer5io/meshery/blob/074493c57e07f8f98fa11664f1873c6264b96889/.github/workflows/build-and-release.yml#L7 
It runs on all pushes to all branches in all forks (e.g. https://github.com/mgfeller/meshery-consul/actions), also upstream of course. The advantage is that it runs before submitting a PR, allowing to catch issues early.

Reply:
That makes much sense. 👍

Reply:
Updated to include * branches on the push event. I can't muster the brainpower to determine whether `master` remains appropriate for pull_request events or if this needs expanded, too. The comment from CodeQL says that branches for pull_request events should be a subset, so it's not clear to me. I've little to no opinion here (on the PR at-large). @mgfeller, please feel at leisure to take this over and get us going in the best direction.